charters/Architecture/charter.md

```diff
@@ -0,0 +1,35 @@
+# DCP Architecture Team
+
+## Description
+
+The Data Coordination Platform (DCP) Architecture Team is responsible for the technical oversight of the DCP, including identifying and addressing *cross-cutting* architectural concerns across DCP projects.
```

Comment:
As I understand the scope of this team, it comprises all (software) technology oversight, including:
* design oversight (aka architecture)
* operational policy oversight (see DevSecOps item below)
* implementation policy oversight, especially end-to-end (see Integration or Technical Best Practices items below)

If correct, the first sentence should capture the full scope.

Suggested revision:
> The Data Coordination Platform (DCP) Architecture Team is responsible for the technical oversight of the DCP, including identifying and addressing *cross-cutting* software design, implementation and operational concerns across DCP projects.   

I also suggest you organize the strategic and tactical responsibilities (below) by these domains.   As workload and complexity increase, some of them may naturally be delegated to sub-teams.